pkg/aks/create.go

```diff
@@ -236,6 +236,11 @@ func createManagedCluster(ctx context.Context, cred *Credentials, workplacesClie
 		managedCluster.APIServerAccessProfile.EnablePrivateCluster = spec.PrivateCluster
 	}
 
+	managedCluster.Identity = &containerservice.ManagedClusterIdentity{
+		TenantID: &cred.TenantID,
+		Type: "SystemAssigned",
+	}
```

Comment:
@MAXxATTAXx hi, thanks for the PR. 

Can you please fix the lint issue?

Reply:
Corrected